lib/active_merchant/billing/gateways/shift4_v2.rb

```diff
@@ -0,0 +1,53 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class Shift4V2Gateway < SecurionPayGateway
+      # same endpont for testing
+      self.live_url = 'https://api.shift4.com/'
+      self.display_name = 'Shift4 V2'
```

Comment:
I asked internally, but I just want to confirm that `Shift4 V2` is the intended display name.

Reply:
We can also rename the old gateway instead

Reply:
It makes sense it should be just "Shift4", Maybe once everything is wired to be the new user-facing version we can change the name of the old one.